compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1778,4 +1778,15 @@ object messages {
       hl"""A class marked with the ${"final"} keyword cannot be extended"""
   }
 
+  case class FailureToEliminateExistential(tp: Types.Type, tp1: Types.Type, tp2: Types.Type, boundSyms: List[Symbols.Symbol])(implicit ctx: Context)
```

Comment:
Simply use `Type` instead of `Types.Type` and `Symbol` instead of `Symbols.Symbol`